src/query.ts

```diff
@@ -67,6 +68,7 @@ export function buildQuery({ tags = [], cursor, owner, sort = DESCENDING_ORDER }
 	return {
 		query: `query {
 			transactions(
+				${ids?.length ? `ids: ${ids.map((id) => `"${id}"`)}` : ''}
```

Comment:
**For the reviewer**
The query works fine for single IDs only and fails for multiple ones. Anyway this PR only adds functionality for `download-file` (single).
See #148 for the latest change in this method

Reply:
This code was de-coupled from the CLI layer https://github.com/ardriveapp/ardrive-core-js/pull/83